roles/install-grsec-kernel/tasks/grub_config.yml

```diff
@@ -25,6 +25,17 @@
       entry exists in the GRUB config file.
   when: grub_grsecurity_menu_entry is not defined
 
+- name: Check whether GRUB DISTRIBUTOR is defined
+  command: grep -q GRUB_DISTRIBUTOR /etc/default/grub
+  register: reg_grub_distributor_check
+  ignore_errors: yes
+  when: ansible_distribution == "Debian"
```

Comment:
This'll need a `changed_when: false`, since it's a read-only task that will never change state.